velox/connectors/hive/HiveConnector.h

```diff
@@ -29,6 +29,11 @@
 
 namespace facebook::velox::connector::hive {
 
+namespace {
+const std::string kHiveConnectorName = "hive";
```

Comment:
Is there a reason to have these here ; they seem to be only used in 1 place each, why not make them default values ?

Reply:
This variable is used in one of the test as well. This is refactored from another header file.

Reply:
I will probably make them class members.